io_scene_nif/modules/nif_import/property/geometry/mesh.py

```diff
@@ -105,7 +105,7 @@ def process_property_list(self, n_block, b_obj):
 
         # run all processors
         for prop in props:
-            NifLog.debug(f"{type(prop)} property found {prop}")
+            NifLog.debug(f"{type(prop)} property found")
```

Comment:
Should we have prop.name so we know what it is, rather than printing out the entire object.

Reply:
I think it is clear from the context, as
a) meshes rarely have duplicate properties
b) properties are rarely named anyway

Reply:
Actually this is fine as my brain meant property type, but that is obviously covered by type(prop).